### DISCUSSION TITLE
No response to GitHub Education verification (1 month)

### DISCUSSION THREAD

**Discussion by finnizzda** [community/community#124859] [GitHub Education]
### Hi everyone,

I went through the verification process about a month ago and haven't received a response regarding acceptance or rejection. I wonder if something is missing, even though I entered the name of my university, uploaded my ID card, added my university email address, entered the correct billing information, and activated 2FA.

I'm not sure if the issue is with the ID card because the example showed that one should only provide the front of the ID with the matriculation number and expiration date, but my card has these details on different sides. A friend of mine from the same university has also been waiting for verification for a month.

I'd appreciate any help.

Greetings

**Comment by nikhilsbuilds** [comment]
Hello, you can use one of your recent fee receipts along with your ID card to verify the details. Hope you get verified soon!
